Makefile_packaging.mk

```diff
@@ -324,20 +329,20 @@ chrootbuild: $(SRPM) $(CALLING_MAKEFILE)
 name=$$repo:$$branch:$$build_number\n\
 baseurl=$${JENKINS_URL:-https://build.hpdd.intel.com/}job/daos-stack/job/$$repo/job/$$branch/$$build_number/artifact/artifacts/centos7/\n\
 enabled=1\n\
-gpgcheck = False\n" >> /etc/mock/default.cfg;                                        \
+gpgcheck = False\n" >> /etc/mock/default.cfg;                                    \
 	    done;                                                                    \
-	    for repo in $(el7_LOCAL_REPOS) $(el7_REPOS); do                          \
+	    for repo in $($(DISTRO_BASE)_LOCAL_REPOS) $($(DISTRO_ID_BASE)_REPOS); do \
 	        repo_name=$${repo##*://};                                            \
 	        repo_name=$${repo_name//\//_};                                       \
 	        echo -e "[$$repo_name]\n\
 name=$${repo_name}\n\
 baseurl=$${repo}\n\
-enabled=1\n" >> /etc/mock/default.cfg;                                               \
+enabled=1\n" >> /etc/mock/default.cfg;                                           \
 	    done;                                                                    \
 	    echo "\"\"\"" >> /etc/mock/default.cfg;                                  \
 	else                                                                         \
 	    echo "Unable to update /etc/mock/default.cfg.";                          \
-            echo "You need to make sure it has the needed repos in it yourself.";    \
+            echo "You need to make sure it has the needed repos in it yourself."; \
```

Comment:
Ditto.

Reply:
Lines 331 and 339 do not show that mis-allignment of the backslash in visual studio code.
Line 334 extends beyond column 80.

Reply:
It's line 345 here I am referring to.  It looks to have had some space before the `\` removed.

Reply:
Not seeing a any \ on line 345, and it lines up with the if on line 314.
Since this is merged, I am looking at current master in Visual Studio Code.